pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/kernelconfig/component.go

```diff
@@ -53,6 +53,13 @@ func (component) Config(ctx components.Context) ([]extensionsv1alpha1.Unit, []ex
 			fmt.Sprintf("%s = %s\n", sysctl.RootMaxBytes, strconv.Itoa(sysctl.RootMaxBytesSetting))
 	}
 
+	if len(ctx.Sysctls) > 0 {
+		newData += "#Custom kernel settings for worker group\n"
+	}
+	for key, value := range ctx.Sysctls {
+		newData += fmt.Sprintf("%s = %s\n", key, value)
+	}
+
```

Comment:
That's a good idea, I think losing the comments on the nodes is fine (it has the benefit that it also reduces the size of to-be-downloaded content). We can still keep the comments in the source code.

Reply:
@ScheererJ Somehow your message was not visible when I responded 👀 Do you think these comments indeed need to present on each and every node? I doubt that anybody is really looking into these files. Also, the order could be made deterministic. But I don't have a strong opinion. If you prefer the current way, then I'm fine as well.

Reply:
Reworked the pull request to use a map instead of the static string. To get reproducible output the lines are sorted by the kernel setting.
Side note: We already had a duplicate entry in our static string, which I removed now.

Please take another look.